### PR TITLE
Update 41 Marcos

### PR DESCRIPTION
**Pull request by GJLazaro**
Polishing Filipino Vocabulary

### CODE REVIEW

File: 41 Marcos [hunk 1]
```diff
@@ -314,13 +314,13 @@
...
-[8:32] At hayagan Siyang nagsalita sa paksang ito. Pagkatapos ay hiniwalay Siya ni Pedro sa isang tabi at nagsimulang pagsabihan Siya.
+[8:32] At hayagan Siyang nagsalita sa paksang ito. Pagkatapos ay ihiniwalay Siya ni Pedro sa isang tabi at nagsimulang pagsabihan Siya.
 [8:33] Pero nang lumingon at tumingin sa mga alagad Niya, pinagsabihan Niya si Pedro sabay sabi, "Lumayas Ka sa harapan ko, Satanas! Dahil di mo iniisip ang mga bagay na patungkol sa Diyos, sa halip ay ang mga bagay na patungkol sa tao.
 [8:34] At nang tawagin ang mga tao kasama ng mga alagad Niya, sinabi Niya sa kanila, "Sinumang gustong sumunod sa likuran Ko, dapat niyang pagkaitan ang sarili, at dapat niyang buhatin ang kanyang krus, at dapat siyang dumikit sa Akin.
 [8:35] Dahil ang sinumang gustong iligtas ang sarili niya ay ipapahamak ito, pero sinumang ipahamak ang sarili niya alang-alang sa Akin at sa Masayang Balita, siya ang magliligtas nito.
-[8:36] Dahil ano'ng mapapala ng isang tao kung mapa sa kanya ang buong mundo, pero ipahamak naman niya ang kaluluwa niya?
+[8:36] Dahil ano ang mapapala ng isang tao kung mapasakanya ang buong mundo, pero ipapahamak naman niya ang kaluluwa niya?
```
Comment (openbibles) [inline 41 Marcos:321]: "ipahamak" - past tense is what is in Greek. "ipapahamak" is in future tense

File: 41 Marcos [hunk 2]
```diff
@@ -166,21 +166,21 @@
...
 [5:24] Kaya't sumama Siya sa kanya. Pero sumunod sa Kanya ang napakaraming tao at nagsiksikan sila sa paligid Niya.
 [5:25] Tapos may isang babae na 12 taon nang dinurugo,
 [5:26] at nagdanas siya ng maraming hirap sa ilalim ng maraming doktor, kaya't naubos nang lahat ang kanyang kabuhayan. Sa kabila nito, wala siyang napakinabangan, sa halip ay nalagay pa siya sa mas malalang katayuan.
 [5:27] Nang marinig ang tungkol kay Jesus, nagpunta siya sa gitna ng napakaraming tao sa likuran at hinipo ang damit Niya.
 [5:28] Nagsabi siya kasi na, "Kung mahawakan ko lang ang mga damit Niya, gagaling ako."
 [5:29] At biglang tumigil ang tagas ng dugo niya at alam niya sa kanyang katawan na magaling na siya mula sa salot.
-[5:30] At nalaman agad ni Jesus sa sarili na may lumabas na kapangyarihan mula sa Kanya. Paglingon sa mga tao ay sinabi Niya, "Sino'ng humipo sa mga damit Ko?"
+[5:30] At nalaman agad ni Jesus sa sarili na may lumabas na kapangyarihan mula sa Kanya. Paglingon sa mga tao ay sinabi Niya, "Sino qng humipo sa mga damit Ko?"
```
Comment (openbibles) [inline 41 Marcos:179]: Can you correct "qng"?

**Comment by GJLazaro** [comment]
noted pastor, good na nakikita nyo yung changes na nireflect ko,
will reflect these changes pastor thanks

**Comment by openbibles** [comment]
Good job, bro! I'm quite impressed sa mga corrections ni Ginang. Thanks a
lot!

Paki lagay nalang sa description next update na it's about applying
linguistic correction or something, para to separate yung work naten. Mine
is translation, yours is polishing the Filipino,, or linguistic
something... Github kasi is like an auditing tool. Kita ng lahat ng tao
yan. Thanks!

**Review by openbibles — APPROVED** [review]

**Pull request merged**